include/mxnet/base.h

```diff
@@ -365,6 +365,10 @@ inline std::ostream& operator<<(std::ostream &out, const Context &ctx) {
 #define MXNET_DESCRIBE(...) describe(__VA_ARGS__ "\n\nFrom:" __FILE__ ":" STRINGIZE(__LINE__))
 #define ADD_FILELINE "\n\nDefined in " __FILE__ ":L" STRINGIZE(__LINE__)
 
+#if MXNET_USE_MKLDNN == 1
+static constexpr size_t MKLDNNAlign = 64;
```

Comment:
I don’t think you need ‘static’ here, do you?

Reply:
if not, maybe put a ‘k’ in front of the constant? seems that way in other places, right?

Reply:
sounds good to me.